backend/director/handler.py

```
@@ -211,10 +231,17 @@ def generate_image_url(self, image_id):
 class ConfigHandler:
     def check(self):
         """Check the configuration of the server."""
-        videodb_configured = True if os.getenv("VIDEO_DB_API_KEY") else False
+        try:
+            videodb_configured = bool(os.getenv("VIDEO_DB_API_KEY"))
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)